pytorch_lightning/trainer/training_loop.py

```diff
@@ -299,6 +303,15 @@ def has_arg(self, *args):
         """Warning: this is just empty shell for code implemented in other class."""
 
     def train(self):
+        # add signal handlers for process kills
+        def _signal_kill_handler(*args):
```

Comment:
won't this interfere with the HPC auto-save signal?

Reply:
I don't know. I think it shouldn't, because this will be triggered only on python interpreter exit (with SIGTERM).